katello_rapidnode.py

```diff
@@ -132,7 +164,7 @@ def child_register(parent, child):
     username, password = get_credentials_children()
     commands = []
     commands.append(
-        'rpm -Uvh http://{0}/pub/katello-ca-consumer-latest.noarch.rpm'
+        'rpm --force -Uvh http://{0}/pub/katello-ca-consumer-latest.noarch.rpm'
```

Comment:
because if user re-runs this script (due to discovered errors, etc), and the RPM is already installed, it will throw a non-zero exit code. This just assures that the katello cert is always reinstalled.